assets/extensions/main.js

```diff
@@ -90,13 +92,37 @@ const Main = () => {
 		},
 	];
 
+	/**
+	 * Filters the featured product toggle.
+	 *
+	 * @param {boolean} hideFeaturedProduct Toggle.
+	 */
+	const hideFeaturedProduct = applyFilters(
+		'sensei-lms.Extensions.hideFeaturedProduct',
+		false
+	);
+
+	/**
+	 * Filters the featured product component.
+	 *
+	 * @param {Object} FeaturedProduct Component.
+	 */
+	const FeaturedProduct = applyFilters(
+		'sensei-lms.Extensions.featuredProduct',
+		FeaturedProductSenseiPro
+	);
```

Comment:
@donnapep I've just read the hooks naming convention proposal. Should I change these to `senseiExtensions...` or it's OK to follow the older convention?

Reply:
Filters renamed in 6ef785ac4f39ef461df092b7065a0e26172f232b. More context in #4816.